### ISSUE TITLE
Convert to the Templatin Module documentation into HTML5 'data-template'

### ISSUE THREAD

**Issue by lguariento** [eXist-db/documentation#247]
It would be nice if all the examples after 'HTML5 Method' title here: http://exist-db.org/exist/apps/doc/templating.xml would be HTML5-ready (and not using the deprecated 'class='), perhaps putting something like 'all the code examples from now on follow this standard' or something like that.

**Comment by duncdrum** [comment]
yes agree, we would need to keep at least one reference to the old syntax since folk are likely to encounter when looking at the code in some apps.

**Comment by lguariento** [comment]
Yep, so I was suggesting to keep everything as it is until the 'HTML5 Method' title, put a little 'heads up' saying that the documentation from then on will use the data-template method, and change the rest of the page accordingly.
